academy/ibc/ibc-token-lifecycle.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "IBC Token Lifecyle"
+order: 99
+description: Inter-chain token transfers
+tag: fast-track
+---
+
+# Lifecycle of an Inter-chain token transfer
+
+One of topics that tends to confuse newcomers to blockchain technology and crypto assets is exact nature of coins and tokens. The terminology implies the existence of physical or logical objects. Newcomers may wonder where there objects are in physical space. 
+
+In reality, there are only ledgers and they are non-local, which is to say they are everywhere and nowhere in particular. Protocols enforce strict rules about the state of the ledgers - rules such as only spending funds one actually has, strictly controlling or fixing the total supply, and so on. 
```

Comment:
Please also connect ledger and protocol agrumentatively. Both sentences read a bit disconnected.